docs/release-notes.rst

```diff
@@ -6,6 +6,68 @@
  Release Notes
 ###############
 
+**************
+ Version 0.31
+**************
+
+Version 0.31.0
+==============
+
+**Release Date:** April 15, 2024
+
+**Breaking Changes**
+
+-  SAML: The underlying SAML implementation has been updated to use a newer, more maintained
+   library. As a result, the master config no longer accepts the `idp_cert_path` field and now
+   requires the `idp_metadata_url` field when using SAML.
```

Comment:
@tara-det-ai @dzhu 
@eecsliu made this change because this release note was not in OSS repo, but in ee repo. could you review this as well?

Reply:
i belive any docs changes in the ee repo just get overwritten at release time

Reply:
@eecsliu replied that these were already moved to OSS docs, i'll mark as resolved